Scripts/Generation_scripts/Diesel.py

```diff
@@ -18,15 +18,23 @@
 import numpy as np
 
 class Diesel():
-    def __init__(self): 
+    def __init__(self,**kwargs): 
         self.size = 1
-        self.location = 'Bahraich'
-        self.CLOVER_filepath = '/***YOUR LOCAL FILE PATH***/CLOVER 4.0'
+        self.location = kwargs.get('location')
+        self.CLOVER_filepath = '.'
         self.location_filepath = self.CLOVER_filepath + '/Locations/' + self.location
         self.generation_filepath = self.location_filepath + '/Generation/'
         self.diesel_filepath = self.generation_filepath + 'Diesel/Diesel inputs.csv'
         self.diesel_inputs = pd.read_csv(self.diesel_filepath,header=None,index_col=0).round(decimals=3)
-
+        # Replace input values with keywords if specified
+        if kwargs.get('diesel_inputs'):
```

Comment:
```
if diesel_input is not None
```

Reply:
If the kwarg is specified as above, then this would default to `None` if the kwarg was not specified.